labs/splice.md

```diff
@@ -99,24 +103,14 @@ We expect that contributors interested in Splice may contribute, for example, ex
 Any group operating a Canton synchronizer may accept and implement new versions of the Splice reference applications at its own discretion. The amulet configuration used by any Canton synchronizer operating group can be made publicly available (visible) by that operating group.
 
 ## Architecture
-The following diagrams show how the Splice applications interact with the Daml platform from Digital Asset.
 
-![SV Node Architecture](./images/Super%20Validator%20Architecture%20Diagram%20for%20Splice-1.jpg)
+The following diagram shows how the Splice applications interact with the Daml platform from Digital Asset.
+
+![SV Node Architecture](./images/Splice-Canton-Global-Synchronizer-2.jpg)
 
-![Validator Node Architecture](./images/Super%20Validator%20Architecture%20Diagram%20for%20Splice-2.jpg)
 
 ### Notes:
+
 The Validator module contains the Wallet module and the Traffic Acquisition module.
 
 The SV App module contains the Amulet smart contract code and the Amulet configuration variables, as well as the Synchronizer Governance app. 
-
-# Initial Committers
-
-- https://github.com/waynecollier-da
-- https://github.com/isegall-da
-
-# Sponsor
-https://github.com/tkuhrt - Tracy Kuhrt, Chair of the Hyperledger Technical Oversight Committee
-
-# Pre-existing repository
-We have an existing private repository. We plan to copy code from that repository to the new Splice repository in the first few weeks after the project has been approved, instead of transferring the repository itself.
```

Comment:
These lines are part of the original template for lab proposals. Ideally we should probably keep these for history and to be consistent with the other lab proposals.